dallinger/experiments/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""A home for all Dallinger Experiments. Experiments should be registered
+with a ``setuptools`` ``entry_point`` for the ``dallinger.experiments`` group."""
```

Comment:
Sure, I didn't mean to get rid of the docstring. I think it might make sense to add a new section to the documentation to cover interacting with experiments at the Python level, but I haven't carefully reviewed what exists already.